src/store/mutations.js

```diff
@@ -96,4 +97,30 @@ export default {
   setLoginTargetLocation(state, location) {
     state.loginTargetLocation = location;
   },
+  createAccount(state) {
+    state.accs.push({
+      idx: state.accountCount,
+      userName: i18n.t('subaccountName', { number: state.accountCount }),
+      color:
+        // eslint-disable-next-line no-bitwise
+        state.accountCount === 1 ? '#00FF9D' : `#${((Math.random() * 0xffffff) << 0).toString(16)}`,
+      shift: Math.floor(Math.random() * 100),
```

Comment:
let's generate them based on account address as I proposed [here](https://github.com/aeternity/aepp-sdk-js/pull/65/files#diff-b088c63374aa5e83e46a7d26a08f04b14d1918a1fdb885f003b9d98e64300816R23-R24), to have them recoverable by mnemonic phrase